appstudio-utils/util-scripts/check-ec-policy.sh

```diff
@@ -4,17 +4,20 @@ set -euo pipefail
 EC_WORK_DIR=${EC_WORK_DIR:-/tmp/ecwork}
 
 DATA_DIR=$EC_WORK_DIR/data
+INPUT_DIR=$EC_WORK_DIR/input
 POLICY_DIR=$EC_WORK_DIR/policies
 
 FORMAT=${1:-pretty}
 
-OPA_QUERY=${OPA_QUERY:-data.main.deny}
+CONFTEST_NAMESPACE=${CONFTEST_NAMESPACE:-main}
 
 [[ ! -d $DATA_DIR ]] && echo "Data dir $DATA_DIR not found!" && exit 1
+[[ ! -d $INPUT_DIR ]] && echo "Input dir $INPUT_DIR not found!" && exit 1
 [[ ! -d $POLICY_DIR ]] && echo "Policy dir $POLICY_DIR dir not found!" && exit 1
 
-opa eval \
+INPUT_FILES=$( find "${INPUT_DIR}" -type f -name "*.json"
```

Comment:
This is to support multiple files as input. `json-input-file` creates the file path and takes the filename as an arg.

Reply:
...which makes me realize it's missing from the ec-policies PR.

Reply:
..later.. actually I think it's better if we don't put the test results into the input. We could still keep the find anyway, but it might be better to remove it since it makes the reader assume there are multiple files.